src/I18n/I18n.php

```diff
@@ -71,7 +71,7 @@ public static function translators(): TranslatorRegistry
         );
 
         if (class_exists(Cache::class)) {
-            static::$_collection->setCacher(Cache::pool('_cake_core_'));
+            static::$_collection->setCacher(Cache::pool('_cake_translations_'));
```

Comment:
We could make this backwards compatible if when `_cake_translations_` wasn't defined we used `_cake_core_` as a fallback with a deprecation warning.